aptos-move/framework/src/natives/aggregator_natives/aggregator_v2.rs

```diff
@@ -229,6 +229,52 @@ fn native_create_aggregator(
     ]))])
 }
 
+
+/***************************************************************************************************
+ * native fun create_unbounded_aggregator<IntElement: copy + drop>(): Aggregator<IntElement>;
+ **************************************************************************************************/
+
+ fn native_create_unbounded_aggregator(
```

Comment:
not super important here, but for my curiosity, do we have ways to unit test these native interfaces (with context)?

Reply:
I think Move unit tests should cover all scenarios?

Reply:
@igor-aptos can we have a test with supported/unsupported types?